### PR TITLE
glTF KHR_lights update

### PR DESCRIPTION
**Pull request by MiiBond**
Updated to match current spec.
https://github.com/KhronosGroup/glTF/pull/1223

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -239,7 +238,12 @@ THREE.GLTFLoader = ( function () {
 
 				case 'spot':
 					lightNode = new THREE.SpotLight( color );
-					lightNode.position.set( 0, 0, 1 );
+					// Handle spotlight properties.
+					var spot = light.spot || {};
+					var innerConeAngle = spot.innerConeAngle ? spot.innerConeAngle : 0;
+					var outerConeAngle = spot.outerConeAngle || Math.PI / 4.0;
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Nit: for consistency with other code here, I would write: 

```js
var innerConeAngle = spot.innerConeAngle !== undefined ? spot.innerConeAngle : 0;
var outerConeAngle = spot.outerConeAngle !== undefined ? spot.outerConeAngle : Math.PI / 4.0;
```

**Review by donmccurdy — CHANGES_REQUESTED** [review]
Thanks, looks good! There are some merge conflicts in various pages though, you may want to cherrypick this to a new branch or rebase.

I believe it will also be necessary to set:

```js
renderer.physicallyCorrectLights = true;
```

... to ensure that `color * intensity` is interpreted as luminous intensity measured in candela.

For now I think we should consider this WIP for testing, and only merge when the KHR_lights spec is nearer to converging.

**Comment by MiiBond** [comment]
Closing to open a clean pull request without conflicts.

File: examples/js/loaders/GLTFLoader.js (hunk 1 above)
Comment (MiiBond) [reply examples/js/loaders/GLTFLoader.js]: Will do.

Also opening a new pull request to clean up the PR.

**Pull request closed without merging**